package.json

```diff
@@ -22,25 +24,24 @@
   ],
   "repository": {
     "type": "git",
-    "url": "https://github.com/tbo47/dagre-es"
+    "url": "git+https://github.com/tbo47/dagre-es.git"
   },
   "dependencies": {
-    "d3": "^7.8.2",
+    "d3": "^7.8.5",
     "lodash-es": "^4.17.21"
   },
   "devDependencies": {
-    "@types/d3": "^7.4.0",
-    "@types/jest": "^29.4.0",
-    "@types/lodash-es": "^4.17.6",
-    "eslint": "^8.36.0",
-    "eslint-plugin-import": "^2.27.5",
-    "prettier": "^2.8.4",
-    "typescript": "^4.9.5",
-    "vitest": "^0.29.2"
+    "@types/d3": "^7.4.3",
+    "@types/jest": "^29.5.8",
+    "@types/lodash-es": "^4.17.11",
+    "eslint": "^8.53.0",
+    "eslint-plugin-import": "^2.29.0",
+    "prettier": "^3.1.0",
+    "typescript": "^5.2.2",
+    "vitest": "^0.34.6"
```

Comment:
By the way, GitHub has a free feature called @dependabot that will automatically make PRs updating dependencies if you want: https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuring-dependabot-version-updates#enabling-dependabot-version-updates

It might be worth enabling if you're feeling lazy and want a robot to update everything for you!